tests/function-controller/e2e/Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: test
+
+test: vendor/k8s.io/kubernetes/test/e2e/framework/create.go
+	@echo '+ Running e2e tests'
+ifdef TEST_ARGS
+	@go test -v ./ -args $(TEST_ARGS)
+else
+	@go test -v ./
+endif
+
+# This patch adds support for few extra types to the 'CreateFromManifests' util function.
+vendor/k8s.io/kubernetes/test/e2e/framework/create.go: hack/k8s_framework_create.patch vendor
+	@echo '+ Patching vendored kubernetes/test/e2e/framework'
+	@patch -p0 < hack/k8s_framework_create.patch
+
+vendor: Gopkg.lock
+	@echo '+ Pulling vendored dependencies'
+	@dep ensure
```

Comment:
this file currently misses the standard targets for our ci system
`ci-pr` `ci-master` `ci-release`